src/DataMiner/SLNetTypes/Info/IDObjects/AlarmTreeID.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Skyline.DataMiner.Net.Messages;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using System;
+
+namespace Skyline.DataMiner.Net
+{
+    /// <summary>
+    /// Represents a reference to one specific alarm tree.
+    /// </summary>
+    [Serializable]
+    public sealed class AlarmTreeID //: DMAObjectRef, IEquatable<AlarmTreeID>
```

Comment:
do we also need to list the properties like ElementID and RootAlarmID ?

Reply:
seems like a good idea yeah